JS-Structure/js-structure_shapes/index.js

```diff
@@ -1,3 +1,5 @@
+import getRandomColor from "./utils/randomColor.js";
```

Comment:
Dear, I'm not sure, but I think the correct sintax here is with the {} around the method/function that is being imported:

import { getRandomColor } from "./utils/randomColor.js";

Reply:
changed ☺️